File: XUCore/additions/NSTabViewAdditions.m
// 
// NSTabViewAdditions.m
// 
// Created by Charlie Monroe
// 
// Copyright (c) 2010-13 Fuel Collective, LLC. All rights reserved.
// 


#import "NSTabViewAdditions.h"
#import "NSViewAdditions.h"

@implementation NSTabView (NSTabViewAdditions)
-(void)setDeepEnabled:(BOOL)flag{
	for (NSTabViewItem *item in [self tabViewItems]){
		[[item view] setDeepEnabled:flag];
	}
}
@end

File: XUCore/additions/NSTabViewAdditions.swift
//
//  NSTabViewAdditions.swift
//  XUCore
//
//  Created by Charlie Monroe on 11/8/15.
//  Copyright © 2015 Charlie Monroe Software. All rights reserved.
//

import Foundation

public extension NSTabView {
	
	/// Sets .enabled on all subviews as `flag`
	public override func setDeepEnabled(_ flag: Bool) {
		for item in self.tabViewItems {
			item.view?.setDeepEnabled(flag)
		}
	}
	
}
